conjur/login/login_controller.py

```diff
@@ -25,6 +25,10 @@ class LoginController:
 
     def __init__(self, ssl_verify, user_password, credential_data, login_logic):
         self.ssl_verify = ssl_verify
+        # For init/login commands, the client (client.py) is not initialized
```

Comment:
move this comment above line `self.ssl_verify = ssl_verify`
and put them between """ and """

Reply:
you got it!